README.md

```diff
@@ -1,6 +1,5 @@
 <div align="center">
 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218262128-27b39f5a-3401-4fbb-bc84-029b7cf5f2c7.png)

- [ ] Kindly put your project name instead of **my-project** to make your guidelines useful :+1:
- [ ] Kindly put **bundle install** instead of **gem install** :blush: